cargo-apk/src/manifest.rs

```diff
@@ -71,6 +73,9 @@ struct AndroidMetadata {
     /// Maps profiles to keystores
     #[serde(default)]
     signing: HashMap<String, Signing>,
+    /// Set up reverse port forwarding before the application launches
+    #[serde(default)]
+    reverse_port_fwd: HashMap<String, String>,
```

Comment:
`adb` calls these `reverse socket connections` and `forward socket connections`, and doesn't use the "port forwarding" terminology in the `reverse` docs - can we do the same?

Otherwise at least spell out `fwd` :grimacing:

Reply:
While at it, I think it's only a minimal stretch to also implement `adb forward` as part of this PR :grimacing:

Reply:
> While at it, I think it's only a minimal stretch to also implement adb forward as part of this PR 😬

I don't think that belong in this PR tbh, it's a different feature that I have no means of testing.

Reply:
> adb calls these reverse socket connections and forward socket connections, and doesn't use the "port forwarding" terminology in the reverse docs - can we do the same?

Regardless of how adb calls it, reverse port forwarding is the normal terminology for this. I can change it but I don't think it makes it more clear.